hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/CommonPbjConverters.java

```diff
@@ -325,6 +325,7 @@ private static <T extends Record, R extends GeneratedMessageV3> R explicitPbjToP
             case UtilPrng -> HederaFunctionality.UTIL_PRNG;
             case TokenReject -> HederaFunctionality.TOKEN_REJECT;
             case TokenAirdrop -> HederaFunctionality.TOKEN_AIRDROP;
+            case TokenCancelAirdrop -> HederaFunctionality.TOKEN_CANCEL_AIRDROP;
```

Comment:
This can also be removed

Reply:
If we remove that the branch will no longer compile